standard/README.md

```diff
@@ -0,0 +1,728 @@
+# Detailed table of contents
+
+<!-- The remaining text is generated by a tool. Do not hand edit -->
```

Comment:
Do we actually need the marker any more? I'd expect we could simplify the code by just making it generate the complete file, including the heading.

Reply:
I left this in for two reasons:

1. It's a note to any contributors not to edit the TOC.
1. If we add more text in the file, it will just work (as it was already tested in the parent folder).

Reply:
Okay - I'm easy either way.